server/game/cards/10-SoD/PeaceAndProsperity.js

```diff
@@ -0,0 +1,19 @@
+const PlotCard = require('../../plotcard.js');
+
+class PeaceAndProsperity extends PlotCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            targetType: 'player',
+            targetController: 'current',
+            effect: [
+                ability.effects.reduceNextMarshalledCardCost(3, card => card.getType() === 'character'),
+                ability.effects.reduceNextMarshalledCardCost(2, card => card.getType() === 'location'),
+                ability.effects.reduceNextMarshalledCardCost(1, card => card.getType() === 'attachment')
```

Comment:
`reduceNextMarshalledCardCost` has a limit of 1 time fixed, so if the plot is recycled and replayed, the effect will no longer be active. Using `reduceFirstMarshalledCardCostEachRound` instead should properly reset the effect each round (and thus keep it active if recycled).

Reply:
Thanks! You've just fixed #1801 as well. :)

Reply:
Ah, good catch!